api/bertymessenger.proto

```diff
@@ -190,7 +190,7 @@ message AppMessage {
     TypeUserReaction = 2;
     TypeGroupInvitation = 3;
     TypeSetGroupName = 4;
-    TypeSetUserName = 5;
+    TypeSetUserName = 5; // FIXME: SetDisplayName ?
```

Comment:
the group also has a DisplayName, if you really want to add "Display", I suggest:
```
TypeSetGroupDisplayName = 4;
TypeSetUserDisplayName = 4;
```

Reply:
or `TypeSetDisplayName` with a `kind` field instead of the two